### PR TITLE
Added TypeScript instructions

### PR DESCRIPTION
**Pull request by pjotrsavitski**
Updated README file to include instructions for using the package with TypeScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,19 @@ This is the solution for you if you're just using `<script>` tags everywhere!
...
+
+```sh
+npm install --save @types/url-regex-safe
+```
+
+After the additional package is installed, it will be possible to import the package and use that in a familiar manner.
+
+```ts
+import urlRegexSafe from "url-regex-safe";
+```
```
Comment (Richienb) [inline README.md:113]:
```suggestion
To use this package with [TypeScript](https://www.typescriptlang.org/), you can install the [`@types/url-regex-safe`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/url-regex-safe) package for type definitions.

```sh
npm install --save-dev @types/url-regex-safe
```
```

Comment (pjotrsavitski) [reply README.md:113]: I'm fine with those changes. One thing that seems to be missing is the closing part for the multi line code block.

File: README.md [hunk 2]
```diff
@@ -98,6 +98,19 @@ This is the solution for you if you're just using `<script>` tags everywhere!
 
 Assuming you are using [browserify][], [webpack][], [rollup][], or another bundler, you can simply follow [Node](#node) usage above.
 
+#### TypeScript
+
+This package will not work with [TypeScript](https://www.typescriptlang.org/) by default and would require installation of [@types/url-regex-safe](https://www.npmjs.com/package/@types/url-regex-safe) in addition the main `url-regex-safe` package.
```
Comment (pjotrsavitski) [inline README.md:103]:
```suggestion
This package will not work with [TypeScript](https://www.typescriptlang.org/) by default and would require installation of [@types/url-regex-safe](https://www.npmjs.com/package/@types/url-regex-safe) in addition to the main `url-regex-safe` package.
```

File: README.md [hunk 3]
```diff
@@ -98,6 +98,12 @@ This is the solution for you if you're just using `<script>` tags everywhere!
 
 Assuming you are using [browserify][], [webpack][], [rollup][], or another bundler, you can simply follow [Node](#node) usage above.
 
+#### TypeScript
+
+To use this package with [TypeScript](https://www.typescriptlang.org/), you can install the [`@types/url-regex-safe`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/url-regex-safe) package for type definitions.
+
+```sh
+npm install --save-dev @types/url-regex-safe
```
Comment (pjotrsavitski) [inline README.md:106]:
```suggestion
npm install --save-dev @types/url-regex-safe
```
```

File: README.md [hunk 4]
```diff
@@ -98,6 +98,12 @@ This is the solution for you if you're just using `<script>` tags everywhere!
...
```
Comment (pjotrsavitski) [inline README.md:107]:
```suggestion
```sh
npm install --save-dev @types/url-regex-safe
```

```

**Comment by pjotrsavitski** [comment]
Not sure why, but GitHub seems to be removing the closing part for the suggested multi line comment.

**Comment by Uzlopak** [comment]
Why not providing the d.ts folder in this repo and referencing it in the package.json in the typings attribute?

Then you would provide the typings automatically without the necessity to install another dependency and if the typings are incomplete or wrong we just have to change them in this package and not somewhere else.

**Comment by Richienb** [comment]
@Uzlopak Sometimes, a maintainer is not interested in maintaining their own type definitions. We can only speculate for now.

**Comment by pjotrsavitski** [comment]
It seems that closing code block is missing. Probably due to some changes during resolving code comments.

**Pull request merged**